components/ImageScroll.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+    <div>
+        <h2 class="image_scroller_title">{{ title }}</h2>
+        <div class="container">
+            <div v-for="i in images" :key="i" class="image_container">
+                <NuxtImg :src="i" />
+            </div>
+            <div class="image_container">
```

Comment:
What is this part for, shouldn't the v-for already handle images[0], images[1], ..., images[n]?

Reply:
Currently, this is only implemented using CSS and within the language, My best solution to the infinite image slideshow was to restart the animation when it reaches the end. This can most definitely be optimized, but this was my basis for the slideshow. The reason the images[0] and images[1] are appended is so that the animation will appear to loop around smoothly

Reply:
Okay will accept this solution for now, but create an issue for it.